autosklearn/pipeline/components/classification/gradient_boosting.py

```diff
@@ -125,34 +93,25 @@ def get_properties(dataset_properties=None):
     @staticmethod
     def get_hyperparameter_search_space(dataset_properties=None):
         cs = ConfigurationSpace()
-        loss = Constant("loss", "deviance")
+        loss = Constant("loss", "auto")
```

Comment:
Could you please order the hyperparameters consistent with the constructor? This will make it easier to follow the code.

Reply:
ok